srcs/parsing/lexer.c

```diff
@@ -90,7 +90,7 @@ static bool	is_empty(char *text)
 	return (true);
 }
 
-static char	*rewrite(char **text, int type)
+char	*rewrite(char **text, int type)

```

Comment:
since now this is no longer static function and it's being used in the executor, can we give it a bit more specific name? like rewrite_env_var ?

Reply:
Ah but it rewrites redirs as well. Then rewrite_envvar_redir ?